src/twisted/application/runner/_exit.py

```diff
@@ -6,35 +6,9 @@
 System exit support.
 """
 
+from enum import IntEnum
 from sys import stdout, stderr, exit as sysexit
-
-from constantly import Values, ValueConstant
-
-
-def exit(status, message=None):
```

Comment:
Moved down after definition of `ExitStatus`, so we don't need a forward reference to it.